src/babel.js

```diff
@@ -283,9 +291,13 @@ export default function({ types: t }) {
           state.hasJSXStyle = null
           state.ignoreClosing = null
           state.file.hasJSXStyle = false
-
           setStateOptions(state)
 
+          // `addDefault` will generate unique id for the scope
+          state.styleComponentImportName = createReactComponentImportDeclaration(
+            state
+          )
```

Comment:
Here you should only generate the unique identifier like I said in my previous comment.

`createReactComponentImportDeclaration` needs to be called only if when necessary (see the comments where I suggest to revert). This way the plugin won't insert imports when the lib is not used.

> Need to find a way to erase the unused imports at the end

Reply:
If you created unique identifer first like `_JSXStyle`, then use it in `addDefault` will still generated a new one `_JSXStyle2` and skip the first one. So it will be useless anyway.

But some utils need to use  the import identifier before exit..like `processTaggedTemplateExpression`, or macro plugin. So I think we have to generate the cerntain id first, and let it be used.

I've add some chanegs to remove the unused imports at the end